takeshape/penny
# Create Customer using REST Admin API

### Test Plan (Steps to test):

1. Create a customer
1. Verify in Shopify that the customer was created correctly

### Checklist:
